packages/lib-classifier/src/components/Classifier/components/ModalTutorial/ModalTutorial.js

```diff
@@ -20,61 +24,52 @@ function storeMapper (stores) {
   }
 }
 
-@inject(storeMapper)
-@observer
-class ModalTutorial extends React.Component {
-  constructor () {
-    super()
+function ModalTutorial ({
+  loadingState = asyncStates.initialized,
+  setModalVisibility = () => true,
+  showModal = false,
+  tutorial,
+  ...props
+}) {
+  const { t } = useTranslation('components')
 
-    this.onClose = this.onClose.bind(this)
-  }
-
-  onClose () {
-    const { setModalVisibility } = this.props
+  function onClose() {
     setModalVisibility(false)
   }
 
-  render () {
-    const { loadingState, showModal, tutorial } = this.props
-    if (loadingState === asyncStates.success && tutorial) {
-      return (
-        <Modal
-          {...this.props}
-          active={showModal}
-          closeFn={this.onClose.bind(this)}
-          title={counterpart('ModalTutorial.title')}
-        >
-          <ResponsiveContext.Consumer>
-            {size => {
-              const width = (size === 'small') ? '100%' : '330px'
-              return (
-                <SlideTutorial
-                  onClick={this.onClose}
-                  pad='none'
-                  width={width}
-                />
-              )
-            }}
-          </ResponsiveContext.Consumer>
-        </Modal>
-      )
-    }
-
-    return null
+  if (loadingState === asyncStates.success && tutorial) {
+    return (
+      <Modal
+        {...props}
+        active={showModal}
+        closeFn={onClose}
+        title={t('ModalTutorial.title')}
+      >
+        <ResponsiveContext.Consumer>
+          {size => {
+            const width = (size === 'small') ? '100%' : '330px'
+            return (
+              <SlideTutorial
+                onClick={onClose}
+                pad='none'
+                width={width}
+              />
+            )
+          }}
+        </ResponsiveContext.Consumer>
+      </Modal>
+    )
   }
-}
 
-ModalTutorial.wrappedComponent.defaultProps = {
-  loadingState: asyncStates.initialized,
-  showModal: false,
-  tutorial: {}
+  return null
 }
 
-ModalTutorial.wrappedComponent.propTypes = {
+ModalTutorial.propTypes = {
   loadingState: PropTypes.string,
   showModal: PropTypes.bool,
   setModalVisibility: PropTypes.func.isRequired,
   tutorial: PropTypes.object
 }
```

Comment:
Not required for this PR, but this reminded me that storybook will generate documentation for component props if we remember to document them in our code.
```js
ModalTutorial.propTypes = {
/** tutorial loading state */
  loadingState: PropTypes.string,
/** show the modal tutorial */
  showModal: PropTypes.bool,
/** call back to set showModal */
  setModalVisibility: PropTypes.func.isRequired,
/** the active tutorial */
  tutorial: PropTypes.object
}
```

It's odd that the tutorial visibility is being set globally. It seems like something that could be handled locally with `useState`. I could be wrong, but I don't think anything else in the classifier reacts to the tutorial being visible.